### ISSUE TITLE
Cannot provide metabat2 abundance table ?

### ISSUE THREAD

**Issue by Valentin-Bio** [RasmussenLab/vamb#230]
Hello developer! thanks for the awesome tool, previously I ran vamb using --jgi flag to specify the metabat2 abundance table made by `jgi_summarize_bam_contig_depths`metabat2 command but seems like with the vamb version 4.1.3 do not accept the --jgi flag anymore: 

`vamb --seed 123 -p 24 --fasta /cl1_noproc.fa --jgi c1_depth.txt --outdir cl1/vamb -t 20 --minfasta 200000`

> 
vamb: error: unrecognized arguments: --jgi depth_files/c1_depth.txt

**Comment by Valentin-Bio** [comment]
solved it by using vamb version 3.0.2 but excluding --seed flag.  why the --jgi flag got excluded from newer versions ? 


bests,


Valentín.

**Comment by jakobnissen** [comment]
Dear @Valentin-Bio

The reason we ever accepted MetaBAT's JGI format is historical. In old versions of Vamb, Vamb used its own BAM abundance estimator. However, we found that the estimates produced by JGI were more accurate, so we recommended users just use that, and pass in the abundances with `--jgi`.
In newer version of Vamb, coverage is estimated using CoverM, which is both faster and more accurate, so there is little reason to recommend using MetaBAT's JGI tool.

More generally, we've moved towards a more standardized workflow: Vamb takes in a contig catalogue and a set of BAM files, then does it thing. It's reasonable to expect that some users want to use Vamb in different ways, e.g. by providing custom generated abundances or what have you. But these users will need to use the underlying Python library `vamb`.
